simple_spearmint/simple_spearmint.py

```diff
@@ -43,7 +47,7 @@ class SimpleSpearmint(object):
 
     """
 
-    def __init__(self, parameter_space, noiseless=False, debug=False):
+    def __init__(self, parameter_space, noiseless=False, debug=False, maximize=False):
```

Comment:
1. Can you make your changes PEP-8 compliant - mainly, just making the max line width 79?  This will prevent a spurious change back in the future.
1. It seems more intuitive to have the flag be `minimize` and have it default to `True`, but that's a nitpick.